saleor/graphql/plugins/dataloaders.py

```diff
@@ -19,3 +21,17 @@ def batch_load(self, keys):
             config_to_template[et.plugin_configuration_id].append(et)
 
         return [config_to_template[key] for key in keys]
+
+
+class PluginManagerByRequestorDataloader(DataLoader):
+    context_key = "plugin_by_requestor"
+
+    def batch_load(self, keys):
+        return [get_plugins_manager(lambda: key) for key in keys]
```

Comment:
I don't think that you need to add `lambda` here. The plugins manager pass this value to the Plugins. `BasePlugin` has typing for this field: `Optional[Callable[[], "Requestor"]]`

Reply:
Please note that `key` has type `Requestor` (implicit, I know, sorry) and thus `lambda: key` has exactly the type `Callable[[], Requestor]`